services/slack/intermediate.go

```diff
@@ -370,6 +374,34 @@ func (t *Transformer) CreateIntermediateUser(user string) {
 	t.Logger.Warnf("Created a new user because the original user was missing from the import files. user=%s", user)
 }
 
+func (t *Transformer) AddFilesToPost(post *SlackPost, skipAttachments bool, slackExport *SlackExport, attachmentsDir string, newPost *IntermediatePost) {
+	if (post.File != nil || post.Files != nil) && !skipAttachments {
+		if post.File != nil {
+			err := addFileToPost(post.File, slackExport.Uploads, newPost, attachmentsDir)
+			if err != nil {
+				t.Logger.WithError(err).Error("Failed to add file to post")
+			}
+		} else if post.Files != nil {
+			for _, file := range post.Files {
+				if file.Name == "" {
+					t.Logger.Warnf("Not able to access the file %s as file access is denied so skipping", file.Id)
+					continue
+				}
+				err := addFileToPost(file, slackExport.Uploads, newPost, attachmentsDir)
+				if err != nil {
+					t.Logger.WithError(err).Error("Failed to add file to post")
+				}
+			}
+		}
+	}
+}
+
+func (t *Transformer) AddAttachmentsToPost(post *SlackPost, newPost *IntermediatePost) (model.StringInterface, []byte) {
+	props := model.StringInterface{"attachments": post.Attachments}
+	propsB, _ := json.Marshal(props)
```

Comment:
```suggestion
	propsByteArray, _ := json.Marshal(props)
```
Isn't there a need to check the error here?

Reply:
Actually, this is a copied code from existing code and they were not checking this before as well